open-api/rest-catalog-open-api.yaml

```diff
@@ -2747,6 +2747,54 @@ components:
         uuid:
           type: string
 
+    AzureCredentials:
+      type: object
+      properties:
+        account-name:
+          type: string
+        account-key:
+          type: string
+        token:
+          type: string
+
+    AwsCredentials:
+      type: object
+      required:
+        - access-key-id
+        - secret-access-key
+      properties:
+        access-key-id:
+          type: string
+        secret-access-key:
+          type: string
+        token:
```

Comment:
AWS refers to this as `session token`, so I think we should use that terminology here: `session-token`